docs/REST API/rest-api.md

```diff
@@ -4923,7 +4924,7 @@ If one or more assigned labels are deleted the profile is considered broken (`br
   "created_at": "2023-03-31T00:00:00Z",
   "updated_at": "2023-03-31T00:00:00Z",
   "checksum": "dGVzdAo=",
-  "labels": [
+  "labels_include_all": [
```

Comment:
Break the existing `labels` key in the response. Change it to `labels_include_all`